tests/test_quoting.py

```diff
@@ -57,9 +57,9 @@ def test_never_quote(quoter):
     do_not_quote = '' .join(["ABCDEFGHIJKLMNOPQRSTUVWXYZ",
```

Comment:
Nitpick: it's not necessary to do join, Python syntax automatically does that with a bunch of strings in brackets.
```python
do_not_quote = (
    "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    "abcdefghijklmnopqrstuvwxyz"
    "0123456789"
    "_.-~"
)
```

Reply:
Feel free to make this change in a separate commit